### ISSUE TITLE
Typing 'r' in inline reply textbox opens/collapses reply box for highlighted item

### ISSUE THREAD

**Issue by valtism** [plibither8/refined-hacker-news#46]
**Issue**

When using keyboard navigation, pressing r opens/closes the inline reply textbox. When typing in the expanded textbox, typing r collapses that textbox and deletes the comment.

If another item is highlighted, typing r opens that textbox and focuses it.

*Aside:* Other navigation still happens when typing, like j triggering next item.

**Proposed fix**

Disable keyboard shortcuts while typing in a reply textbox.

*Aside:* May be good to cache replies so that if the reply box is closed and reopened the reply is still there. Don't want people accidentally throwing away long replies.

**OS / Browser**

Windows 10, Chrome.

**Comment by plibither8** [comment]
Hey Dan, thanks for opening this issue.

This bug was discussed on #44 and I've pushed the fix and release of it. The Firefox add on has been updated successfully but unfortunately the Chrome extension's update is "pending review" which is notoriously known to take almost a week to be successfully available on the webstore.

I will update you when the extension gets accepted on the Chrome Webstore.

Cheers!
